README.md

```diff
@@ -263,5 +263,26 @@ EXAMPLE
   $ spot validate api.ts
 ```
 
+_See code: [build/cli/src/commands/validate.js](https://github.com/airtasker/spot/blob/v0.2.13/build/cli/src/commands/validate.js)_
+
+## `spot tag SPOT_CONTRACT`
+
+Generate a version tag based on a Spot contract
+
+```
+USAGE
+  $ spot tag SPOT_CONTRACT
+
+ARGUMENTS
+  SPOT_CONTRACT  path to Spot contract
+
+OPTIONS
+  -h, --help             show CLI help
+  --algorithm=algorithm  digest algorithm used to generate the hash
```

Comment:
What are the options?

Reply:
hey  -  will remove this option. See comment below.